common/config/rush/build-cache.json

```diff
@@ -1,12 +1,12 @@
 {
-  "buildCacheEnabled": false,
+  "buildCacheEnabled": true,
   // Follow instructions at
   //   https://rushjs.io/pages/maintainer/build_cache/#user-authentication
   // to authenticate.
   "cacheProvider": "azure-blob-storage",
   "azureBlobStorageConfiguration": {
-    "storageAccountName": "azsdkjsrush",
-    "storageContainerName": "rush-build-cache",
+    "storageAccountName": "azuresdkartifacts",
+    "storageContainerName": "azure-sdk-for-js-rush-cache",
```

Comment:
We have an existing azure-sdk-for-js container can this go under that or will it splat overtop whatever is already in there?

Reply:
I want to have a life cycle management rule of, for example, delete blobs that have not been modified for 90 days.  Could we add a rule to `azure-sdk-for-js`? Another concern is that these build cache blobs are not human readable names and there can be a lot of them, so they may impact the browsing experience of other blobs in the same containers.

Reply:
If it needs to be an isolated container I'm good with this.